misc/services/mobile-webui/mobile-webui-frontend-testing/tests/spec/picking/picking_catchWeight.spec.js

```diff
@@ -217,6 +234,33 @@ test('EAN13 with prefix 29 and not matching product', async ({ page }) => {
     });
 });
 
+// noinspection JSUnusedLocalSymbols
+test('EAN13 with a standard prefix', async ({ page }) => {
```

Comment:
isn't picking_productBasedAggregation.spec.js a better place for this test?
see `Filter by EAN13`